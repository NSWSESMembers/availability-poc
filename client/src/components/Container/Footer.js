import PropTypes from 'prop-types';
import React from 'react';
import { View } from 'react-native';
import styles from './styles';

const Footer = ({ children }) => (
  <View style={styles.containerFooter}>
    {children}
  </View>
);

Footer.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Footer;
