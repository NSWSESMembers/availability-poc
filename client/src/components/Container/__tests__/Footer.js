import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';

import Footer from '../Center';

test('renders correctly', () => {
  const tree1 = renderer.create(
    <Footer>
      <Text>test</Text>
    </Footer>,
  ).toJSON();
  expect(tree1).toMatchSnapshot();
});
