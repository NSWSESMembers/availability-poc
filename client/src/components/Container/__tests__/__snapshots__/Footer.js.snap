// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    test
  </Text>
</View>
`;
